### ISSUE TITLE
Publishing tables fails

### ISSUE THREAD

**Issue by ralfschulze** [asciidoctor/asciidoctor-confluence#30]
Trying to publish a file with a table:

table.adoc:
```
== Demo Table

|===
| Col1 | Col2 | Col3 | Col4

| Text1 | Text2 | Text3 | Text4
|===
```

`asciidoctor-confluence --host https://<host> --spaceKey <key> --title <title> --username <username> --password <password> table.adoc`

I get the errors:

```
/home/rschulze/.gem/ruby/2.7.0/gems/faraday-0.9.2/lib/faraday/options.rb:153: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
/home/rschulze/.gem/ruby/2.7.0/gems/faraday-0.9.2/lib/faraday/options.rb:153: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
/home/rschulze/.gem/ruby/2.7.0/gems/faraday-0.9.2/lib/faraday/options.rb:153: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
/home/rschulze/.gem/ruby/2.7.0/gems/faraday-0.9.2/lib/faraday/options.rb:153: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
/home/rschulze/.gem/ruby/2.7.0/gems/faraday-0.9.2/lib/faraday/options.rb:153: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
/home/rschulze/.gem/ruby/2.7.0/gems/faraday-0.9.2/lib/faraday.rb:68: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
/home/rschulze/.gem/ruby/2.7.0/gems/faraday-0.9.2/lib/faraday/rack_builder.rb:55: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
/home/rschulze/.gem/ruby/2.7.0/gems/faraday-0.9.2/lib/faraday/options.rb:62: warning: Capturing the given block using Proc.new is deprecated; use `&block` instead
/home/rschulze/.gem/ruby/2.7.0/gems/json-1.8.6/lib/json/common.rb:155: warning: Using the last argument as keyword parameters is deprecated
An error occurred, the page has not been created because:
Error parsing xhtml: Unexpected close tag </colgroup>; expected </col>.
 at [row,col {unknown-source}]: [10,10]
```

Confluence version is 7.2.0

**Comment by r-vanooyen** [comment]
same problem will happen with adocs "+" which results in an html5 <br> tag. 

solution will be to set the backend format to xhtml5 (works for me localy with br's)

**Comment by mojavelinux** [comment]
resolved by #26
